### ISSUE TITLE
Missing successful global message when user sets additional Payment Method as default

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13720]
**Describe the bug**
- Screen reader users wouldn't be able to tell if they have changed a Payment Method as the new default.

**Tell us the version of Spartacus**
- Library version: 4.0.0

**To Reproduce**
1. Go to My Account > Payment Details
2. Add 2 different Payment Methods or more.
3. Open any Screen Reader (VO, NVDA, JAWS).
4. Switch Payment Method from additional to new defaults.
5. Listen to Screen Reader feedback
6. Hear the absence of confirmation a change has happened.

**Expected behavior**
- Screen reader users will hear a successful confirmation message as they set a Payment Method as new default. E.g _"Additional Payment Method successfully set as new default"._
- For `MSG_TYPE_CONFIRMATION` use `aria-live="polite" aria-atomic="true"`

**Screenshots**

Message suggestion in the screenshot below, please validate phrasing with @Manurod2018 or @vekterdude

![p2021-09-14_20-54-34](https://user-images.githubusercontent.com/15113219/133353913-b8cd9fd5-8b35-42b5-8238-880115a8d1e5.jpg)

**Comment by KateChuen** [comment]
Tested on develop branch - chrome. 

![paymentVO](https://user-images.githubusercontent.com/45409515/144272519-da4dcfc2-037a-4a7e-813d-768799b7fc5e.png)


More extensive testing (on other screenreaders and devices) will be done with RC.
